pkg/graphviz/traceflow.go

```diff
@@ -219,6 +222,19 @@ func genOutput(graph *gographviz.Graph, isSingleCluster bool) string {
 	return str
 }
 
+func getTraceflowStatusMessage(tf *opsv1alpha1.Traceflow) string {
+	switch tf.Status.Phase {
+	case opsv1alpha1.Failed:
+		return getWrappedStr(fmt.Sprintf("traceflow %s failed: %s", tf.Name, tf.Status.Reason))
+	case opsv1alpha1.Running:
+		return getWrappedStr(fmt.Sprintf("traceflow %s is running...", tf.Name))
+	case opsv1alpha1.Pending:
+		return getWrappedStr(fmt.Sprintf("traceflow %s is pending...", tf.Name))
+	default:
+		return "Unknown traceflow status, please check whether Antrea is running and traceflow is enabled."
```

Comment:
doesn't this message need to be wrapped by `"`?

Reply:
Added getWrappedStr.